### PR TITLE
Dev/new UI

### PR DESCRIPTION
**Pull request by Jungyoonlim**
I added lots of refinements -- new summary copy btn, new loading spinner, formatted time, fixed responsiveness for times, animated icons etc.

### CODE REVIEW

File: .env.sample
Comment (sam5pell) [inline .env.sample:1]: we probs want to keep the sample file in so people know what env vars they need

File: components/Conversations/Components/Summary.tsx [hunk 1]
```diff
@@ -31,9 +34,39 @@ const Summary: React.FC<SummaryProps> = ({ transcript, onSummaryGenerated, conve
     setGeneratedSummary(existingSummary || null)
   }, [conversationId])
 
+  const buttons = useTranscriptButtons({
+    message: generatedSummary ?? '',
+    // buttonTypes: ['Copy', 'Share', 'Save', 'SendFeedback']
+    buttonTypes: ['Copy'] // TODO: Add other buttons
+  })
```
Comment (sam5pell) [inline components/Conversations/Components/Summary.tsx:41]: Doesn't seem like this is actually used anywhere. Can delete?

**Comment by sam5pell** [comment]
Start time for active conversation is updating to match the current time
image

**Comment by sam5pell** [comment]
Besides the timestamp issue I mentioned above everything looks good :)

File: components/Conversations/Components/Summary.tsx (hunk 1 above)
Comment (Jungyoonlim) [reply components/Conversations/Components/Summary.tsx:53]: yep!

File: .env.sample
Comment (Jungyoonlim) [reply .env.sample:1]: ah I see! I'll add this in

**Comment by Jungyoonlim** [comment]
all of the issues mentioned should be resolved! let me know how it looks :) and thank you for being patient with me -- wasn't so sure about how this works

File: components/Conversations/Detail/Icon/DetailIcons/TranscriptionDisabled/VideoPlayIcon.tsx
Comment (sam5pell) [inline components/Conversations/Detail/Icon/DetailIcons/TranscriptionDisabled/VideoPlayIcon.tsx:1]: FYI most of our icons come from the iconsax library - you can see the icon name and variant (ex. bold in this case) in figma! so for this one you would just do
```
import { VideoPlay } from 'iconsax-react'

<VideoPlay variant="Bold" />
```
```
![image](https://github.com/user-attachments/assets/76131572-81c5-4315-8083-816445c2c63e)

**Review by sam5pell — APPROVED** [review]
Looks good! Feel free to merge :)

**Pull request merged**